src/sentry/db/models/fields/node.py

```diff
@@ -113,19 +113,14 @@ def data(self):
             return self._node_data
 
         rv = {}
-        if self.field is not None and self.field.wrapper is not None:
-            rv = self.field.wrapper(rv)
+        if self.wrapper is not None:
+            rv = self.wrapper(rv)
         return rv
 
     def bind_data(self, data, ref=None):
         self.ref = data.pop("_ref", ref)
-        self.ref_version = data.pop("_ref_version", None)
-        if (
-            self.field is not None
-            and self.ref_version == self.field.ref_version
-            and ref is not None
-            and self.ref != ref
-        ):
```

Comment:
If the ref_function is always the same and you are likely going to remove it, does this check on ref_version and ref still make sense ?
I am trying to figure out if there is a good reason for preserving ref and ref_version.

Reply:
I wasn't actually considering removing it, just commenting it's always the same function passed in here. I left the ref checking logic unchanged (apart from not needing the first `self.field` check anymore).

Longer term i'm not sure whether it's worth preserving ref and ref_version (what's the chance of node id collision?), but I'd prefer not to tackle this right now.